jobs/eam-integrations/scripts/workday_netsuite_integration.py

```diff
@@ -0,0 +1,64 @@
+from argparse import ArgumentParser
+import logging
+from workday_netsuite.api.workday import WorkDayRaaService
+from api.util import Util
+
+ 
+class WorkdayToNetsuiteIntegration():
```

Comment:
I have absolutely 0 context on what you're doing here, but I wanted to make sure you've [seen this](https://docs.getcensus.com/destinations/available-destinations/netsuite#supported-objects), which may be an easier way to upload data to Netsuite.